js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/invertedIndex/FieldsDropdown.tsx

```diff
@@ -41,8 +45,12 @@ const MultiValueLabel = (
 export const FieldsDropdown = ({ field }: { field: IndexFormFieldProps }) => {
   const [formikField] = useField<InvertedIndexValuesType[]>(field.name);
   const { setCurrentFieldData } = useInvertedIndexContext();
+  console.log("formikField", formikField.value);
```

Comment:
Same here..

Reply:
yeah, left it in by mistaske, removed now